android/src/main/java/com/thebluealliance/androidclient/gcm/notifications/BaseNotification.java

```diff
@@ -25,12 +26,16 @@
 import com.thebluealliance.androidclient.listitems.ListElement;
 import com.thebluealliance.androidclient.models.StoredNotification;
 import com.thebluealliance.androidclient.receivers.NotificationChangedReceiver;
+import com.thebluealliance.androidclient.viewmodels.GenericNotificationViewModel;
+import com.thebluealliance.androidclient.viewmodels.ViewModelRenderer;
 
 import java.text.DateFormat;
 import java.util.Calendar;
 import java.util.Date;
 
-public abstract class BaseNotification extends ListElement {
+import io.nlopez.smartadapters.views.BindableLayout;
+
+public abstract class BaseNotification<VIEWMODEL> extends ListElement implements ViewModelRenderer<VIEWMODEL, Void> {
```

Comment:
Are there any cases where we need to render one model into different kinds of `ViewModel`s? If that's something we need to worry about, the empty parent class may be the best way to go. If not, I don't think it'll make a difference.